haskell-language-server.cabal

```diff
@@ -59,12 +65,14 @@ library
     , haskell-lsp == 0.20.*
     , hie-bios >= 0.4
     , hslogger
+    , lens
     , optparse-simple
     , process
     , regex-tdfa >= 1.3.1.0
     , shake >= 0.17.5
     , text
     , transformers
+    , unix
```

Comment:
ugh, this one make it unbuildable in windows 😄

Reply:
what workarounds are there?

Reply:
sorted, copied the approach from hie-plugin-api